unittests/resources/checks/bad/relative_imports.py

```diff
@@ -0,0 +1,4 @@
+from . import invalid_iterable
```

Comment:
Can you put a small comment at the beginning of this fail explaining its purpose?